_posts/2020-11-17-quarkiverse.adoc

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: Quarkiverse Hub - Home of Community Extensions
+date: 2020-11-17
+tags: extensions quarkiverse
+synopsis: Entering a world of extensions
+author: gastaldi
+---
+
+Quarkus is a Kubernetes Native Java stack tailored for the OpenJDK HotSpot and GraalVM, crafted from the best of breed Java libraries and standards, as well as an ever growing ecosystem of its extensions. At the beginning we were accepting all the contributed extensions in the core Quarkus repository. Eventually though it grew too large and lead to a maintenance overhead. In addition to that, in some cases it's simply not always sensible to include an extension into the core Quarkus repository.
```

Comment:
I don't think it's a good idea to lead with this. It sounds to me like you are saying "there was an internal maintenance issue that the team needed to fix so we moved some stuff around. Here's a blog post on how to deal with the move"

I think it would be better to lead with some Tl;Dr; that excites the community. For example, something like: 

"Today we are announcing the Quarkiverse! A single repository where any contributor can host their Quarkus extension. Hosting your extension in the Quarkiverse makes it available to all Quarkus users through the Quarkus tooling, and over time we'll start to add other features like <insert some features we are willing to talk publicly about>. So read on to discover more and find out how you can make your extension available to all Quarkus developers today!"

Obviously that's a bit rough around the edges, and I'm likely not using all the correct terminology. But hopefully you get the idea.

Reply:
I agree this is much better, let me give it another try